openmoc/thisown.i

```diff
@@ -107,6 +107,23 @@
         universes[i][j][k].thisown = False
 %}
 
+/* A Lattice owns the memory for each Universe it contains */
+%pythonappend Lattice::updateUniverse %{
+  # SWIG 3
+  if 'universe' in locals():
+    universe = locals()['universe']
```

Comment:
This confuses me. From my interpretation of what ``locals()`` does, this line should do nothing